e2e/consulacls/consul-acls-manage.sh

```diff
@@ -0,0 +1,375 @@
+    doActivate
+  fi
+
+  echo "=== Enable: DONE ==="
+
+  # show the status of all the agents
+  echo "---> token file is ${token_file}"
+  consul_http_token=$(cat "${token_file}")
+  export CONSUL_HTTP_TOKEN="${consul_http_token}"
+  echo "export CONSUL_HTTP_TOKEN=${CONSUL_HTTP_TOKEN}"
+  doStatus
+}
+
+function doDisable() {
+  if [ ! -f "${token_file}" ]; then
+    echo "DISABLE: token file does not exist, did bootstrap ever happen?"
+    exit 1
+  else
+    echo "DISABLE: token file exists, will deactivate ACLs"
+    doDeactivate
+  fi
+
+  echo "=== Disable: DONE ==="
+
+  # show the status of all the agents
+  unset CONSUL_HTTP_TOKEN
+  doStatus
+}
+
+function doActivate() {
+  echo "=== Activate ==="
+
+  stopConsul
+
+  # Upload acl-enable.hcl to each Consul Server agent's configuration directory.
+  for server in ${servers}; do
+    echo " activate: upload acl-enable.hcl to ${server}::acl.hcl"
+    doSCP "consulacls/acl-enable.hcl" "${user}" "${server}" "/tmp/acl.hcl"
+    doSSH "${server}" "sudo mv /tmp/acl.hcl ${consul_configs}/acl.hcl"
+  done
+
+  # Restart each Consul Server agent to pickup the new config.
+  for server in ${servers}; do
+    echo " activate: restart Consul Server on ${server} ..."
+    doSSH "${server}" "sudo systemctl start consul"
+    sleep 1
+  done
+
+  sleep 10
+
+  startConsulClients
+
+  sleep 10
+
+  echo "=== Activate: DONE ==="
+}
+
+function stopNomad {
+  echo "=== Stop Nomad agents ==="
+  # Stop every Nomad agent (clients and servers) in preperation for Consul ACL
+  # bootstrapping.
+  for server in ${servers}; do
+    echo " stop Nomad Server on ${server}"
+    doSSH "${server}" "sudo systemctl stop nomad"
+    sleep 1
+  done
+
+  for linux_client in ${linux_clients}; do
+    echo " stop Nomad Client on ${linux_client}"
+    doSSH "${linux_client}" "sudo systemctl stop nomad"
+    sleep 1
+  done
+
+  echo "... all nomad agents stopped"
+}
+
+function startNomad {
+  echo "=== Start Nomad agents ==="
+  # Start every Nomad agent (clients and servers) after having Consul ACL
+  # bootstrapped and configurations set for Nomad.
+  for server in ${servers}; do
+    echo " start Nomad Server on ${server}"
+    doSSH "${server}" "sudo systemctl start nomad"
+    sleep 1
+  done
+
+  # give the servers a chance to settle
+  sleep 10
+
+  for linux_client in ${linux_clients}; do
+    echo " start Nomad Client on ${linux_client}"
+    doSSH "${linux_client}" "sudo systemctl start nomad"
+    sleep 3
+  done
+
+  # give the clients a long time to settle
+  sleep 30
+
+  echo "... all nomad agents started"
+}
+
+function stopConsul {
+  echo "=== Stop Consul agents ==="
+  # Stop every Nonsul agent (clients and servers) in preperation for Consul ACL
+  # bootstrapping.
+  for server in ${servers}; do
+    echo " stop Consul Server on ${server}"
+    doSSH "${server}" "sudo systemctl stop consul"
+    sleep 1
+  done
+
+  for linux_client in ${linux_clients}; do
```

Comment:
It looks like we're not stopping/starting the Windows clients. Does this cut the Window clients out from having Consul workloads once they're activated on the servers? (That's probably ok for now but we'll want to fix that when we get the other workload tests added for Windows.)